eng/testing/RunnerTemplate.sh

```diff
@@ -252,18 +204,47 @@ if [[ "$(uname -s)" == "Linux" && $test_exitcode -ne 0 ]]; then
   if [[ "$core_name_uses_pid" == "1" ]]; then
     # We don't know what the PID of the process was, so let's look at all core
     # files whose name matches core.NUMBER
-    echo Looking for files matching core.* ...
-    for f in core.*; do
-      [[ $f =~ core.[0-9]+ ]] && print_info_from_core_file "$f" "dotnet" && copy_core_file_to_temp_location "$f" && rm "$f"
+    echo "Looking for files matching core.*"
+    for f in $(find . -name "core.*"); do
+      [[ $f =~ core.[0-9]+ ]] && move_core_file_to_temp_location "$f"
     done
-  elif [ -f core ]; then
-    echo found a dump named core in $EXECUTION_DIR !
-    print_info_from_core_file "core" "dotnet"
-    copy_core_file_to_temp_location "core"
-    rm "core"
+  fi
+
+  if [ -f core ]; then
+    move_core_file_to_temp_location "core"
+  fi
+
+  total_dumps=$(find $HELIX_DUMP_FOLDER -name "*.dmp" | wc -l)
+  
+  echo ----- start ===============  XUnitLogChecker Output =====================================================
+  xunitlogchecker_exit_code=0
+  if [ $total_dumps -gt 0 ]; then
+    
+    echo "Total dumps found in $HELIX_DUMP_FOLDER: $total_dumps"
+    xunitlogchecker_file_name="$HELIX_CORRELATION_PAYLOAD/XUnitLogChecker.dll"
+    dotnet_file_name="$RUNTIME_PATH/dotnet"
+
+    if [ ! -f $dotnet_file_name ]; then
+      echo "'$dotnet_file_name' was not found. Unable to run XUnitLogChecker."
+      xunitlogchecker_exit_code=1
+    elif [ ! -f $xunitlogchecker_file_name ]; then 
+      echo "'$xunitlogchecker_file_name' was not found. Unable to print dump file contents."
+      xunitlogchecker_exit_code=2
+    else
+      cmd="$dotnet_file_name --roll-forward Major $xunitlogchecker_file_name --dumps-path $HELIX_DUMP_FOLDER"
+      echo "$cmd"
+      $cmd
```

Comment:
This is driving me crazy. It works when tested locally in my Ubuntu WSL, the command gets executed. But in the CI it's saying $dotnet does not exist in that location (`$RUNTIME_PATH/dotnet`).
I'm adding more verbosity to see what's going on.

Reply:
Actually the one failing is the Windows version. Same action item for me.